apps/website/lib/config.dart

```diff
@@ -2,6 +2,8 @@ import 'package:flutterkaigi_2025_website/config_stub.dart'
     if (dart.library.js_interop) './config_web.dart';
 import 'package:flutterkaigi_2025_website/text.dart' show Language;
 
+final eventDate = DateTime.parse('2025-11-13T10:00:00+0900').toLocal();
```

Comment:
```suggestion
final _eventDate = DateTime.parse('2025-11-13T10:00:00+0900').toLocal();
```

L73あたりにある event.dateを使って欲しいかも

```diff
final event = (
  title: 'FlutterKaigi 2025',
-  date: DateTime.parse('2025-11-13T10:00:00+0900').toLocal(),
+  date: _eventDate,
```